docs/tutorial/window-customization.md

```diff
@@ -267,3 +267,15 @@ draggable areas.
 [overlay-css-env-vars]: https://github.com/WICG/window-controls-overlay/blob/main/explainer.md#css-environment-variables
```

Comment:
One more thing is that we would want to put your new addition above line 264 where all the reference links are!